### DISCUSSION TITLE
CUDA memory error then access denied/app can't run issue on Windows 10

### DISCUSSION THREAD

**Discussion by zho0080** [NVlabs/instant-ngp#571] [Q&A]
Previously when I ran testbed.exe the first time on Anaconda Prompt, it actually got to the stage of running: '01:07:01 ERROR Uncaught exception' and had a white window pop up and then automatically close.
Then the conda prompt stated it failed with error CUDA_ERROR_OUT_OF_MEMORY but after I changed the aabb value in the transform json file from 16 originally to 8, I started getting the pop-up 'This app can't run on your PC' and below 'To find a version for your PC, check with the software publisher.' when I try to run: instant-ngp\build\testbed.exe --scene data/fox
Using the fox image set. And the Anaconda Prompt displays 'Access is denied.' 
Even when I lower to 4, 1, or change aabb value back to 16, and re-run, it still displays 'Access is denied'.

Are there methods of resolving this issue or getting testbed.exe to run again without error? 

By the way, the PC (Windows 10 Pro) is running on an NVIDIA GeForce GTX 1070 with dedicated video memory of 8192 MB, and total available graphics memory of 24554 MB.

**Comment by jaj6422** [comment]
Similar issue here. I'm trying to run this
C:\ngp\instant-ngp\build\instant-ngp.exe --scene data/<name>
Anaconda displays 
'Access is denied.'
With a pop-up "This app can't run on your pc"
